earth_enterprise/src/common/khCache.h

```diff
@@ -92,41 +91,7 @@ class khCache {
     }
     return false;
   }
-  void CheckListInvariant(void) {
```

Comment:
We should arguably leave this function in since we might want to enable it for testing at some point.

Reply:
Ah - got it, just remove the invocations, not the implementation.